app/core/Tracker2/FacebookPixelTracker.js

```diff
@@ -0,0 +1,58 @@
+import BaseTracker from './BaseTracker'
+
+function loadFacebookPixel () {
+  !function(f,b,e,v,n,t,s)
+  {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
+    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
+    n.queue=[];t=b.createElement(e);t.async=!0;
+    t.src=v;s=b.getElementsByTagName(e)[0];
+    s.parentNode.insertBefore(t,s)}(window, document,'script',
+    'https://connect.facebook.net/en_US/fbevents.js');
+
+  fbq('init', '514962702046652');
+  fbq('track', 'PageView')
+}
+
+export default class FacebookPixelTracker extends BaseTracker {
+  constructor (store) {
+    super()
+
+    this.store = store
+  }
+
+  async _initializeTracker () {
+    this.watchForDisableAllTrackingChanges(this.store)
+
+    // Facebook pixels are currently tracked via Segment, which is enabled for all teachers so do not
```

Comment:
Is this related to another instance of the tracker that I was able to find?

https://github.com/codecombat/codecombat/blob/859cced6dc1c5828c271a58a0af39a56e9247675/app/views/core/CreateAccountModal/teacher/TeacherRolePanel.coffee#L40

Reply:
It's the same tracker but I looked for it's loading and can't find it being loaded anywhere so I think it's legacy.  That or it depends on the segment facebook pixel that is loaded.